module/VuFind/src/VuFind/Controller/AbstractRecord.php

```diff
@@ -614,8 +615,8 @@ public function smsAction()
                 $sms->text($view->provider, $view->to, null, $body);
                 $this->flashMessenger()->addMessage('sms_success', 'success');
                 return $this->redirectToRecord();
-            } catch (MailException $e) {
-                $this->flashMessenger()->addMessage($e->getMessage(), 'error');
+            } catch (SMSException $e) {
```

Comment:
I think it's okay to leave this as a MailException, since that's more general than SMSException and will catch both types -- this way, if either type is thrown, the correct behavior will occur. I think it's possible that sending an SMS will lead to a MailException being thrown, if something goes wrong in lower-level mail code.

Reply:
Sure thing!